### PR TITLE
Move package to psr7-sessions org

### PR DESCRIPTION
**Pull request by malukenho**
- [x] Change namespaces?
- [x] Composer
- [x] Read.me (text, links)
- [x] Unit Tests
- [x] Enable build

### CODE REVIEW

**Comment by malukenho** [comment]
Need to enable composer to install this package on packagist after merge it.

File: docs/configuration.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ $sessionMiddleware = new SessionMiddleware(
     $cookieBlueprint,
     new \Lcobucci\JWT\Parser(),
     1200, // session lifetime, in seconds
-    new \PSR7Session\Time\SystemCurrentTime(), // Current time provider implementation using current time system
+    new \PSR7Session\Storageless\Time\SystemCurrentTime(), // Current time provider implementation using current time system
```
Comment (Ocramius) [inline docs/configuration.md]: Just noticed: `s/current time system/current system time`

File: test/StoragelessTest/Session/LazySessionTest.php [hunk 2]
```diff
@@ -145,8 +145,8 @@ public function testIsEmpty()
     {
         $this->wrappedSessionWillBeLoaded();
 
-        $this->wrappedSession->expects($this->at(0))->method('isEmpty')->willReturn(true);
-        $this->wrappedSession->expects($this->at(1))->method('isEmpty')->willReturn(false);
+        $this->wrappedSession->expects(self::at(0))->method('isEmpty')->willReturn(true);
```
Comment (Ocramius) [inline test/StoragelessTest/Session/LazySessionTest.php]: Thanks for updating these!

**Comment by Ocramius** [comment]
Question: is it worth to move to 7.1-only?

**Comment by Ocramius** [comment]
Note that I opened #2 to provide better test coverage.

File: composer.json [hunk 3]
```diff
@@ -31,12 +32,12 @@
...
+            "PSR7Session\\Storageless\\": "src/Storageless"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "PSR7SessionTest\\": "test/PSR7SessionTest"
+            "PSR7SessionTest\\Storageless\\": "test/StoragelessTest"
         }
     },
     "extra": {
```
Comment (lcobucci) [inline composer.json]: We should also change the branch-alias

Comment (Ocramius) [inline composer.json]: As discussed with @malukenho, do we want to proceed with 3.0, or do we want to restart from 1.0? /cc @necromant2005

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-[![Packagist](https://img.shields.io/packagist/vpre/ocramius/psr7-session.svg)](https://packagist.org/packages/ocramius/psr7-session)
+[![Build Status](https://travis-ci.org/psr7-sessions/storageless.svg)](https://travis-ci.org/psr7-sessions/storageless)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/psr7-sessions/storageless/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/psr7-sessions/storageless/?branch=master)
+[![Packagist](https://img.shields.io/packagist/v/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
+[![Packagist](https://img.shields.io/packagist/vpre/psr7-sessions/storageless.svg)](https://packagist.org/packages/psr7-sessions/storageless)
 
 **PSR7Session** is a [PSR-7](http://www.php-fig.org/psr/psr-7/)
 [middleware](https://mwop.net/blog/2015-01-08-on-http-middleware-and-psr-7.html) that enables
```
Comment (lcobucci) [inline README.md]: Maybe mention that the `storageless` package is not only about session usage in PSR-7 based applications but also without storing anything (it sounds a bit stupid when put like this but is just to differentiate from the other possible packages of the org)

**Comment by Ocramius** [comment]
@lcobucci can you merge in #2 if you got time?

**Comment by lcobucci** [comment]
@Ocramius sure

File: composer.json (hunk 3 above)
Comment (lcobucci) [inline composer.json]: Both are ok but I prefer to not discard the project history. I mean, it's the same package but just moved to an organization...

Comment (Ocramius) [inline composer.json]: Makes sense. At this point, I'd say that it's better to move the past repo then, with all its issues.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,32 +1,6 @@
 This is a list of changes/improvements that were introduced in PSR7Session
 
-## 2.0.0
+## 1.0.0
 
-This release contains backwards compatibility breaks with previous releases.
```
Comment (Ocramius) [inline CHANGELOG.md]: Will have to revert this pre-merge

**Comment by Ocramius** [comment]
Note: branch is to be kept, but we'll likely move https://github.com/Ocramius/PSR7Session, so that all issues/PR numbers are also ported correctly

File: composer.json (hunk 3 above)
Comment (lcobucci) [inline composer.json]: 👍 lets do that

Comment (Ocramius) [inline composer.json]: Let's finish up the PR, then we move the repo and merge this as a new PR into it :-)

**Comment by Ocramius** [comment]
Last notification: deleting this repo, moving ocramius/psr7-session here.

**Pull request closed without merging**